snuba/web/wsgi.py

```diff
@@ -1,6 +1,8 @@
+from snuba.core.initialize import initialize_snuba
 from snuba.environment import setup_logging, setup_sentry
 
 setup_logging()
 setup_sentry()
+initialize_snuba()
```

Comment:
not related to this ticket but I'm starting to think all of these should be in the `initialize` function